silvestre-cli/src/commands/apply.rs

```
@@ -0,0 +1,228 @@
+        .ok_or_else(|| CliError::Custom(format!("missing parameter: {}", key)))?
+        .parse::<u32>()
+        .map_err(|_| CliError::Custom(format!("{} must be an unsigned integer", key)))
+}
+
+fn convert_to_silvestre(img: &image::DynamicImage) -> Result<SilvestreImage> {
+    let rgba = img.to_rgba8();
+    let (width, height) = img.dimensions();
+    let pixels = rgba.to_vec();
+
+    SilvestreImage::new(pixels, width, height, ColorSpace::Rgba)
+        .map_err(|e| CliError::Silvestre(e))
```

Clippy (1.95.0)

[warning] redundant closure

(warning)